lesson05/assignment/index.html

```diff
@@ -8,8 +8,40 @@
         <title>Assignment 05 - CSS Grid</title>
     </head>
     <body>
-        <main class="main">
-          
-        </main>
+        <div class="grid">
+        <header>
+            <p class="header__text">Olivia, Off the Grid</p></header>
+        <nav>
```

Comment:
You would want to give your `<nav>` a class of `nav`, but otherwise the BEM is spot on.